packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-import-value.yaml

```diff
@@ -0,0 +1,6 @@
+Resources:
+  Bucket1:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName:
+        !ImportValue SomeSharedValue
```

Comment:
That's what I'm talking about 🙂 Love this test!

Reply:
gotcha, thanks 😄